README.md

```diff
@@ -128,7 +128,7 @@ The simplest way is to use `kubectl run`.
 kubectl run spartakus \
     --image=gcr.io/google_containers/spartakus-amd64:v1.0.0 \
     -- \
-    volunteer --cluster-id=$(uuidgen -r)
+    volunteer --cluster-id=$(uuidgen)
```

Comment:
On OSX uuidgen doesn't have `-r` and on linux `-r` is default.

https://linux.die.net/man/1/uuidgen
"By default uuidgen will generate a random-based UUID if a high-quality random number generator is present."